### PR TITLE
Removes experimental prefix from work_pools routes

### PR DESCRIPTION
**Pull request by desertaxle**
Removes /experimental prefix from work_pools routes in anticipation of work pools GA release.

### CODE REVIEW

**Comment by zhen0** [comment]
@marichka-offen - for you when ready

**Comment by marichka-offen** [comment]
Looks good to me, I just want to make sure the backend goes first or very soon after for both Cloud and OSS so we don't have broken UI. I'm working on some big fixes and don't want to be blocked 
@desertaxle do you have more information about when this change hits the backend?

**Review by brandonreid — APPROVED** [review]

**Comment by desertaxle** [comment]
> Looks good to me, I just want to make sure the backend goes first or very soon after for both Cloud and OSS so we don't have broken UI. I'm working on some big fixes and don't want to be blocked @desertaxle do you have more information about when this change hits the backend?

I have PRs open for Cloud and OSS to make a similar update. Once they're ready to merge, I'll coordinate with you to find a time to merge them all in very close succession.

**Comment by marichka-offen** [comment]
@desertaxle sounds good! Let's not merge this PR just now, then. As soon as you give me green light, I'll merge it, release orion-design and update its version in Cloud and OSS UIs

**Review by cicdw — APPROVED** [review]

**Pull request merged**
